src/praisonai/praisonai/bots/agentmail.py

```
@@ -111,7 +111,12 @@ def __init__(
         
         # Session manager for conversation state
         self.config = config or BotConfig()
-        self._session = BotSessionManager(self.config)
+        try:
+            from praisonaiagents.session import get_default_session_store
+            _store = get_default_session_store()
+        except Exception:
```

Ruff (0.15.17)

[warning] Do not catch blind exception: `Exception`

(BLE001)